setup.py

```diff
@@ -34,7 +34,7 @@
       include_package_data=True,
       zip_safe=True,
       install_requires=[
-        "pymongo",
+        "pymongo[encryption]<4.9",
```

Comment:
I'm ok with this for now, to get this MR merged.  But we probably should make it work on 4.9 next, before making a release :|

Reply:
Agreed.